### PR TITLE
fixes issue#927

### PR DESCRIPTION
**Pull request by jimccfun**
fixes issue [https://github.com/opensds/opensds/issues/927](https://github.com/opensds/opensds/issues/927)

### CODE REVIEW

**Comment by jimccfun** [comment]
@wisererik @leonwanghui @anvithks @liuqinguestc

File: src/app/business/block/file-share-detail/file-share-detail.component.html [hunk 1]
```diff
@@ -178,30 +178,20 @@ <h3>{{i18n.keyID['sds_block_volume_base_info']}}</h3>
                             </form-item>
                             <form-item [required]="true" label="{{label.user}}">
                                 <div class="ui-grid-row volume-item" *ngFor="let AclsItem of aclsItems;index as i">
-                                    <!-- <p-dropdown [options]="availabilityip" formControlName="{{'user'+i}}" [style]="{'min-width':96}"></p-dropdown> -->
                                     <div class="ui-grid-col-9" style="margin-bottom: -10px">
-                                        <input type="text" pInputText formControlName="{{'userInput'+i}}" [style]="{'width':150}" />
-                                        <div class="ui-message-container" *ngIf="createAclsFormGroup.controls['userInput'+i] && !createAclsFormGroup.controls['userInput'+i].valid&&createAclsFormGroup.controls['userInput'+i].dirty">
+                                        <input type="text" pInputText formControlName="{{'userInput'+i}}" [style]="{'width':150}" (keyup)="userIpChnage(i)"/>
```
Comment (anvithks) [inline src/app/business/block/file-share-detail/file-share-detail.component.html]: fix typo userIpChnage() -> userIpChange()

File: src/app/business/block/file-share-detail/file-share-detail.component.ts [hunk 2]
```diff
@@ -241,11 +226,22 @@ export class FileShareDetailComponent implements OnInit{
...
                     this.acls.push(acl);
+                    if(dialog){
+                        this.checkAclsIp.push(item.accessTo);
+                    }
                 }
             })
             this.selectedAcls = [];
         })
     }
+    userIpChnage(index){
```
Comment (anvithks) [inline src/app/business/block/file-share-detail/file-share-detail.component.ts]: Fix typo

File: src/app/business/block/fileShare.component.html [hunk 3]
```diff
@@ -66,29 +66,20 @@
             </form-item>
             <form-item [required]="true" label="{{I18N.keyID['sds_fileShare_acl_user']}}">
                 <div class="ui-grid-row volume-item" *ngFor="let AclsItem of aclsItems;index as i">
-                    <!-- <p-dropdown [options]="availabilityip" formControlName="{{'user'+i}}" [style]="{'min-width':96}"></p-dropdown> -->
                     <div class="ui-grid-col-9" style="margin-top: -10px">
-                        <input type="text" pInputText formControlName="{{'userInput'+i}}"/>
-                        <div class="ui-message-container" *ngIf="createAclsFormGroup.controls['userInput'+i] && !createAclsFormGroup.controls['userInput'+i].valid&&createAclsFormGroup.controls['userInput'+i].dirty">
+                        <input type="text" pInputText formControlName="{{'userInput'+i}}" (keyup)="userIpChnage(i)"/>
```
Comment (anvithks) [inline src/app/business/block/fileShare.component.html]: fix typo

File: src/app/business/block/fileShare.component.ts [hunk 4]
```diff
@@ -180,9 +182,31 @@ export class FileShareComponent implements OnInit{
...
+        this.FileShareAclService.getFileShareAcl().subscribe((res)=>{
+            let str = res.json();
+            str.forEach(item=>{
+                if(item.fileshareId == selectedFileShare.id){
+                    this.checkAclsIp.push(item.accessTo);
+                }
+            })
+        })
+    }
+    userIpChnage(index){
```
Comment (anvithks) [inline src/app/business/block/fileShare.component.ts]: fix typo.

Comment (jimccfun) [reply src/app/business/block/fileShare.component.ts]: I have just modified it. Please check it

File: src/app/business/block/fileShare.component.html (hunk 3 above)
Comment (jimccfun) [reply src/app/business/block/fileShare.component.html]: I have just modified it. Please check it

File: src/app/business/block/file-share-detail/file-share-detail.component.ts (hunk 2 above)
Comment (jimccfun) [reply src/app/business/block/file-share-detail/file-share-detail.component.ts]: I have just modified it. Please check it

File: src/app/business/block/file-share-detail/file-share-detail.component.html (hunk 1 above)
Comment (jimccfun) [reply src/app/business/block/file-share-detail/file-share-detail.component.html]: I have just modified it. Please check it

Comment (anvithks) [reply src/app/business/block/file-share-detail/file-share-detail.component.html]: I still see that the function name is not changed.  
(keyup)="userIpChnage(i)" -> (keyup)="userIpChange(i)"

File: src/app/business/block/fileShare.component.html (hunk 3 above)
Comment (anvithks) [reply src/app/business/block/fileShare.component.html]: I still see that the function name is not changed.  
(keyup)="userIpChnage(i)" -> (keyup)="userIpChange(i)"

File: src/app/business/block/fileShare.component.ts (hunk 4 above)
Comment (anvithks) [reply src/app/business/block/fileShare.component.ts]: I still see that the function name is not changed.  
(keyup)="userIpChnage(i)" -> (keyup)="userIpChange(i)"

**Comment by anvithks** [comment]
LGTM

**Comment by anvithks** [comment]
@wisererik Please review.

**Comment by anvithks** [comment]
@wisererik Please review this.

**Comment by anvithks** [comment]
@skdwriting Please approve so I can merge.

**Pull request merged**